Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -392,9 +392,9 @@ public void DeleteItem(BaseItem item, DeleteOptions options, BaseItem parent, bo
                 // Add this flag to GetDeletePaths if required in the future
                 var isRequiredForDelete = true;
 
-                foreach (var fileSystemInfo in item.GetDeletePaths().ToList())
+                foreach (var fileSystemInfo in item.GetDeletePaths())
                 {
-                    if (File.Exists(fileSystemInfo.FullName))
+                    if (fileSystemInfo.IsDirectory ? Directory.Exists(fileSystemInfo.FullName) : File.Exists(fileSystemInfo.FullName))
```

Comment:
Can we wrap it or change into several statements? I am not a fan of such complex ternaries...

Reply:
I don't think this ternary is particularly confusing, and I think further nesting wouldn't be a better solution, but if you still think it should be changed I suppose I can. Do you have a good suggestion?